### PR TITLE
Simplify client generation

### PR DESCRIPTION
**Pull request by CatTail**

### CODE REVIEW

**Comment by sivy** [comment]
Looks good to me :+1:  A second +1 gets it merged.

**Comment by devdazed** [comment]
:+1:

**Comment by devdazed** [comment]
I would bump the version to 1.0.0 or something though.. Since this will break the existing API. Plus it's about time we are on 1.x I don't see the API changing much anymore.

**Comment by CatTail** [comment]
> Since this will break the existing API

where?

**Comment by sivy** [comment]
It will break users who are currently using the existing constructor.

## 

Steve - steveivy@gmail.com

File: lib/statsd.js [hunk 1]
```diff
@@ -223,4 +223,5 @@ Client.prototype.close = function(){
     this.socket.close();
 }
 
+exports = module.exports = Client;
 exports.StatsD = Client;
```
Comment (CatTail) [inline lib/statsd.js:227]: @sivy This line will backward compatible with old style constructor usage

Comment (sivy) [inline lib/statsd.js:227]: I don't have access to test it right now - if you can show that both constructors continue to work I'm ok merging this.

## 

Steve - steveivy@gmail.com

Comment (devdazed) [inline lib/statsd.js:227]: ah, yes, I guess you're right. I misread that

Comment (CatTail) [inline lib/statsd.js:227]: Merge please :)

**Comment by CatTail** [comment]
Please publish a new version in npm.

**Pull request merged**